### PR TITLE
Fixes #711: Email Sending Utility

### LINKED ISSUE #711: Add Mail utility function
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behaviour:**
There is no mail utility end point.

**Expected behaviour:**
There should be a mail utility. Use cases are when a user authenticates then there should be a verification mail sent  to him with the passed token. Also after verifying the token then a greeting mail should be sent to user.

__Yes__ working on it.

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes #711 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

- Email Sending Utility
-

### CODE REVIEW

File: backend/blueprint/api/utils/mail.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    'Couldn\'t find the username specified'))
+        try:
+            msg = Message(
+                subject=message.subject,
+                sender=app.config['MAIL_USERNAME'],
+                recipients=[message.receipent],
+                body=message.body)
+            Mail(app).send(msg)
+        except Exception as e:
+            print(e)
```
Comment (gabru-md) [inline backend/blueprint/api/utils/mail.py]: You are already using it in sending Response, so I guess we can remove thie `print` statement?
What say?

File: backend/blueprint/api/run.py [hunk 2]
```diff
@@ -9,16 +9,21 @@
...
 app.config['BASE_DIR'] = os.path.dirname(os.path.abspath(__file__))
 app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://%(user)s:%(pw)s@%(host)s:%(port)s/%(db)s' % config.POSTGRES
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
 app.config['SECRET_KEY'] = config.POSTGRES['secret']
 app.config['DEBUG'] = config.DEBUG
+app.config.from_object('config.mailConfig.MailConfig')
 
 db.init_app(app)
 migrate = Migrate(app, db)
+mail = Mail(app)
```
Comment (djmgit) [inline backend/blueprint/api/run.py]: Do we require to instantiate a Mail object here? We are using Mail object in mail.py right?

**Comment by yashLadha** [comment]
Done @djmgit @gabru-md Please review it again.

**Comment by yashLadha** [comment]
Yes @djmgit currently assuming that the username is the email as the email will be unique identity. May be we can add email field in the user model also.

**Pull request merged**